assignments/objects.js

```diff
@@ -18,27 +18,58 @@ const example = {
 }
 
 // Write your intern objects here:
-
-
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@PushSubscription.edu",
+  "gender": "F"
+}
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
```

Comment:
```suggestion
  "gender": "M",
  speak: function() {
    return `Hello, my name is ${this.name}`
  }
```

Reply:
More what they had in mind, don't think it got the lecture time it deserved.  See if you can the one for Antonietta's fn. with a similar process.